04-cs-fundamentals/readme.md

```diff
@@ -1,51 +0,0 @@
-Apr 04  | [Understanding function calls](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/04-cs-fundamentals/classroom/Understanding%20function%20calls.md)
-Apr 11  |  Solving problems [using Hash tables](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/04-cs-fundamentals/classroom/Using%20Hash%20Tables.md)
-Apr 25  | [Interview Process](classroom/Interview-prorocess.md) <br />[Whiteboarding tips](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/04-cs-fundamentals/classroom/Whiteboarding-Tips.md)
-May 02  | [Interview Process](classroom/Interview-prorocess.md) <br />[Whiteboarding tips](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/04-cs-fundamentals/classroom/Whiteboarding-Tips.md)
-**Week of May 05** | **Mock interviews**
-May 09  | [Hash Tables](https://adaacademy.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=731632dc-fe3b-4cc3-9f0b-aa4500686a3e)<br>[Hash Tables Lesson](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/04-cs-fundamentals/classroom/hash-tables.md)<br>[Linked Lists](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/04-cs-fundamentals/classroom/Introduction%20to%20Linked%20Lists.md) <br>[Linked List video](https://adaacademy.hosted.panopto.com/Panopto/Pages/Viewer.aspx?id=bf9edf20-8a83-409f-af82-aa46000f81ee)
-May 16 |  **Community Afirmation Activity**
-May 23 | [Recursion, Merge sort](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/04-cs-fundamentals/classroom/Introduction%20to%20Recursion.md)
-May 30  | [Interview prep](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/04-cs-fundamentals/classroom/Whiteboarding-Tips.md)
-**Week of June 3rd** | **Internship placement interviews**
-Jun 20 | [Introduction to software design](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/04-cs-fundamentals/classroom/Software%20Design.md)
-
-Left for self-reading and learning: [Sorting](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/04-cs-fundamentals/classroom/Sorting.md)
-
-## Internship - Cohort 10
-We'll be meeting on Thursday afternoons from 1:00 pm to 5:00 pm in the Ada Auditorium. During 9 of these Thursdays, we'll chat about CS Fundamentals. Here's the  [internship calendar](https://github.com/Ada-Developers-Academy/daily-curriculum/blob/master/cohort_schedules/C10_internship.md) for Thursdays at Ada.
-
-Here are the topics we'll cover:
-- [Linked Lists](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/04-cs-fundamentals/internship/Linked%20Lists.md) 
-  - [Slide deck](https://drive.google.com/open?id=0B__DV26QHsH4bFczWXBXdGtHYkE) 
-  - [Homework](https://github.com/Ada-Developers-Academy/textbook-curriculum/blob/master/04-cs-fundamentals/homeworks.md#linked-lists)
```

Comment:
Is this list of resources still valuable, either to students or to you?

Reply:
I moved where they are linked.  I'm also moving from slides to textbook.